defaults/aws_config.yaml

```diff
@@ -47,3 +44,5 @@ ami_id_db_oracle: ''
 use_preinstalled_scylla: true
 
 backup_bucket_backend: 's3'
+backup_bucket_location: 'manager-backup-tests-us-east-1'
```

Comment:
@ShlomiBalalis is it ok that we backup all regions to the same bucket?
Is this intended? don't we have tests that backup each region to its own bucket?

@fgelcer you were involved with this, right?